modules/react/button/lib/PrimaryButtonKing.tsx

```diff
@@ -0,0 +1,423 @@
+          background: themePrimary.main,
+          icon: themePrimary.contrast,
+          label: themePrimary.contrast,
+        },
+        hover: {
+          background: themePrimary.dark,
+        },
+        active: {
+          background: themePrimary.darkest,
+        },
+        focus: {
+          background: themePrimary.main,
+        },
+        disabled: {
+          background: themePrimary.main,
+        },
+      };
+
+    case 'inverse':
+      return {
+        default: {
+          background: colors.frenchVanilla100,
+          icon: colors.blackPepper400,
+          label: colors.blackPepper400,
+          labelData: colors.blackPepper400,
+        },
+        hover: {
+          background: colors.soap300,
+          icon: colors.blackPepper500,
+          label: colors.blackPepper500,
+          labelData: colors.blackPepper500,
+        },
+        active: {
+          background: colors.soap300,
+          icon: colors.blackPepper500,
+          label: colors.blackPepper500,
+          labelData: colors.blackPepper500,
+        },
+        focus: {
+          background: colors.frenchVanilla100,
+          border: colors.blackPepper400,
+          icon: colors.blackPepper400,
+          label: colors.blackPepper400,
+          labelData: colors.blackPepper400,
+          focusRing: focusRing(
+            {
+              separation: 1,
+              innerColor: colors.blackPepper500,
+              outerColor: colors.frenchVanilla100,
+            },
+            theme
+          ),
+        },
+        // Identical to inverse 'default' styles. ButtonContainer will set opacity to 40%
+        disabled: {
+          background: colors.frenchVanilla100,
+          icon: colors.blackPepper400,
+          label: colors.blackPepper400,
+          labelData: colors.blackPepper400,
+        },
+      };
+  }
+};
+
+function getIconColorSelectorsNew(
+  {
+    canvas: {
+      palette: {primary: themePrimary},
+    },
+  }: EmotionCanvasTheme,
+  color: string,
+  fill?: boolean
+): CSSObject {
+  return {
+    '&:focus span, &:hover span, & span': {
+      '.wd-icon-fill, .wd-icon-accent, .wd-icon-accent2': {
+        fill: color,
+      },
+      '.wd-icon-background': {
+        fill: fill ? color : undefined,
+      },
+      '.wd-icon-background ~ .wd-icon-accent, .wd-icon-background ~ .wd-icon-accent2': {
+        fill: fill
+          ? color === themePrimary.contrast
+            ? themePrimary.main
+            : themePrimary.contrast
+          : color,
+      },
+    },
+  };
+}
+
+export interface PrimaryButtonKingProps extends Themeable, GrowthBehavior {
+  /**
+   * The variant of the PrimaryButton.
+   * @default undefined
+   */
+  variant?: 'inverse';
+  /**
+   * There are four button sizes: `extraSmall`, `small`, `medium`, and `large`.
+   * If no size is provided, it will default to `medium`.
+   *
+   * @default 'medium'
+   */
+  size?: ButtonSizes;
+  /**
+   * The data label of the Button.
```

Comment:
Not being familiar with buttons, I had no idea what the "data label of the Button" was. I found code that referenced it in a visual test story and found the story that referenced it in Storybook. This probably means we need a better description.

Perhaps something like:
```suggestion
   * The data label is additional information about the button. The `children` of the Button will
   * appear in bold font while the `dataLabel` will not be emphasized as bold. The `dataLabel` will
   * be part of a button's accessible name appended after `children`. Useful to ancillary data
   * associated with the button, but not the prominent label. If `dataLabel` contents are
   * left-to-right characters, and the `children` are RTL characters, the button may show contents
   * out of order.
   *
```

Reply:
So my latter comment made me wonder if we need specially handling around the data label: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi